conformance/utils/suite/suite.go

```diff
@@ -90,6 +90,7 @@ type ConformanceTestSuite struct {
 	Applier           kubernetes.Applier
 	SupportedFeatures map[SupportedFeature]bool
 	TimeoutConfig     config.TimeoutConfig
+	SkipTests         []ConformanceTest
```

Comment:
Nit: Can we use k8s sets util to avoid looping through this list every time to check for the presence of a test name? (https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1beta1/validation/common.go#L56). Related comment, it would probably be more efficient to just track the test name instead of the full test definition.

Reply:
👍 Done